ocs_ci/ocs/ocp.py

```diff
@@ -159,6 +159,7 @@ def exec_oc_cmd(
             return yaml.safe_load(out)
         return out
 
+    @retry(CommandFailed, tries=3, delay=5, backoff=1)
```

Comment:
Will retry work if the actual command (argument  cmd_list) fail ? 
Retry is needed only if the debug pod is not created due to some issue.

Reply:
cmd fails dcoz debug pod was not created so we are retrying to create debug pod